tendrl/commons/objects/definition/master.yaml

```diff
@@ -37,6 +37,8 @@ namespace.tendrl:
           - TendrlContext.integration_id
           - Cluster.node_configuration
           - Cluster.conf_overrides
+      pre_run:
+        - tendrl.objects.Node.atoms.ValidNodeDetails
```

Comment:
If suggest we change the atom name from "ValidNodeDetails" to "IsNodeTendrlManaged" 

which gives us an atom to do all checks we can if a certain node is Tendrl managed